terms.rdf

```diff
@@ -82,6 +82,24 @@
         <rdfs:subPropertyOf rdf:resource="http://www.w3.org/2000/01/rdf-schema#comment" />
     </rdf:Property>
 
+    <rdf:Property rdf:about="http://cldf.clld.org/v1.0/terms.rdf#position" dc:type="generic-property">
+        <rdfs:comment rdf:parseType="Literal">
+            <p>
+                A position of an item in a series or sequence of items. While integer is recommended
+                as datatype, any datatype that can be totally ordered (and this order is transparent
+                e.g. alphabetic order for strings) can be used. A column for this property may also
+                be list-valued (e.g. to implement multi-level orderings), in which case the typical
+                order for tuples is assumed.
```

Comment:
Proposal for reformulating this as I had to read it multiple times. Feel free to ignore:

"A position represents the placement of an item in a series or sequence of items. Although an integer is the recommended datatype, any datatype that supports a total ordering (where the order is transparent, such as alphabetic order for strings) is acceptable. It is also possible to have a list-valued column for this property, which can be useful for implementing multi-level orderings. In such cases, the typical order for tuples is assumed."